src/components/sq-modal-profile-cache/sq-modal-profile-cache.component.tsx

```
@@ -0,0 +1,236 @@
+  type: SocialNetworksAvailables
+  onSubmit: () => void
+}
+
+const SqModalProfileCache = ({ open, onClose, onSubmit, socialNetworkObject, fieldsCache, type }: Props) => {
+  const { t } = useTranslation('sqModalProfileCache')
+  const [width, setWidth] = useState(window.innerWidth)
+  const [height, setHeight] = useState(window.innerHeight)
+  const [state, setState] = useState(
+    fieldsCache?.reduce(
+      (acc, field) => ({
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)